content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,32 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.7 [2020-03-27]
+
+### Features
+
+- Add option to display dashboards in [light mode](/v2.0/visualize-data/dashboards/control-dashboard/#toggle-dark-mode-and-light-mode).
+- Add [shell `completion` commands](/v2.0/reference/cli/influx/completion/) to the `influx` CLI.
+  specified shell (`bash` or `zsh`).
+- Make all `pkg` resources unique by `metadata.name` field.
```

Comment:
Scott, for line 18, how did you word this in the package/templates doc and what doc best to link to here?

Reply:
I don't know you need to link anything here. This is regarding the act JSON/YAML file that the `pkg` command outputs which currently we don't document. You could change `pkg` to "template," but there's still some bike-shedding going on here that will likely change the entire nomenclature around templates/packages (there's some cool stuff coming).

Reply:
Wanted to be sure we didn't have new pkg/templates docs we wanted to introduce for beta 7